docs/contributing.md

```diff
@@ -51,6 +27,6 @@ There are three different pusher app environments and the credentials for connec
 
 The [Debug Console](https://dashboard.pusher.com/apps/550921/console/realtime_messages) in the Pusher Dashboard will let you see events coming in and going out of each pusher app. 
```

Comment:
Per [Slack thread](https://dosomething.slack.com/archives/CA0N5FXND/p1554143890017100), I couldn't figure out how to get access to see this.